scaffolds/streamlit_on_cloudrun/image_classification/train_and_deploy.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>In the line, ".... but we can simply use Python modules to define the visual and behavior of it." Is visuals refering to the web page elements? If so, maybe rephrase as "but we can simply use Python modules to define and configure the the web page elements".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Updated. Thanks!</p>